<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Layouts</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="layoutsListWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addPushButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removePushButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Keyboard Shape</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="standardRadioButton">
           <property name="text">
            <string>Standard Keyboard</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="MatrixRadioButton">
           <property name="text">
            <string>Matrix Keyboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Keyboard Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="IsoRadioButton">
           <property name="text">
            <string>ISO</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="AnsiRadioButton">
           <property name="text">
            <string>ANSI</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="analyzePushButton">
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="comparePushButton">
        <property name="text">
         <string>Compare</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionAboutKLA"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuLayout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/Images/document-new.png</normaloff>:/images/Images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/Images/system-log-out.png</normaloff>:/images/Images/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/Images/symbol_help.png</normaloff>:/images/Images/symbol_help.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAboutKLA">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/Images/symbol_information.png</normaloff>:/images/Images/symbol_information.png</iconset>
   </property>
   <property name="text">
    <string>About KLA</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
